### DISCUSSION TITLE
has the git status desync bug been resolved ?

### DISCUSSION THREAD

**Discussion by bilal-houari** [zed-industries/zed#17272] [Feature Requests]
zed version is 0.150.4

I've looked for bug reports regarding this, the newest I could find was from a month ago and it's marked as completed.
I've also checked the changelog for the preview releases newer that 0.150.4 but can't find anything related.

![image](https://github.com/user-attachments/assets/7499b3e4-5ece-4921-af68-6b8c9b5ca1e9)

**Comment by JosephTLyons** [comment]
This PR likely fixed the Git status sync issue you're talking about:

- https://github.com/zed-industries/zed/pull/16926

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
